renovate.json5

```diff
@@ -1,10 +1,51 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
-    "config:base",
+    "config:base"
   ],
-  ignorePaths: [
-      "src/jvmTest/resources/",
+  enabled: true,
+  enabledManagers: [
+    "gradle",
+    "gradle-wrapper",
+    "github-actions",
+  ],
+  // Will auto-merge directly, without a PR, if tests pass - else, makes a PR.
+  // Must add Renovate to 'Allow specified actors to bypass required pull requests'
+  // in branch protection rule
+  automergeType: "branch",
+  platformAutomerge: true,
+  ignoreTests: false,
```

Comment:
It's false by default: https://docs.renovatebot.com/configuration-options/#ignoretests
Do we need to be explicit here?

Reply:
I prefer being explicit (because who knows if the default changes...) but it can be removed if you want.

```suggestion
```